src/test/java/software/amazon/smithy/lsp/ext/DocumentTest.java

```diff
@@ -47,6 +51,27 @@ public void detectPreamble() throws Exception {
         assertTrue(preamble.isBlankSeparated());
     }
 
+    @Test
+    public void detectPreambleV2() throws Exception {
+        Path baseDir = Paths.get(Document.class.getResource("models/v2").toURI());
+        Path preambleModel = baseDir.resolve("preamble.smithy");
+        List<String> lines = Files.readAllLines(preambleModel);
+        DocumentPreamble preamble = Document.detectPreamble(lines);
+
+        assertEquals("ns.preamble", preamble.getCurrentNamespace().get());
+        assertEquals(new Position(4, 0), preamble.getNamespaceRange().getStart());
+        assertEquals(new Position(4, 21), preamble.getNamespaceRange().getEnd());
+        assertEquals("2.0", preamble.getIdlVersion().get());
```

Comment:
Nit: we should also consider adding tests for `2` since that is what most of the 2.0 docs use.

Reply:
Added in [0ed9978](https://github.com/awslabs/smithy-language-server/pull/61/commits/0ed9978ed9b3d58df77fb1af80310514f8d352bd).